policyengine_us/parameters/gov/states/de/tax/income/deductions/age_eligibility.yaml

```
@@ -0,0 +1,14 @@
+description: Delaware provides the aged additional personal credits amount to filers at or above this age.
+values:
+  2021-01-01: 65
```

Per the law and tax forms. Please be careful.
```suggestion
  2021-01-01: 60
```